pkg/driver/vz/vm_darwin.go

```diff
@@ -108,10 +110,34 @@ func startVM(ctx context.Context, inst *limatype.Instance, sshLocalPort int) (*v
 					filesToRemove[pidFile] = struct{}{}
```

Comment:
This line was not effective until this PR, because above `defer`:
```golang
	filesToRemove := make(map[string]struct{})
	defer func() {
		for f := range filesToRemove {
			_ = os.RemoveAll(f)
		}
	}()
```
has passed before `filesToRemove[pidFile] = struct{}{}` has effect.
Because I changed to wait `waitSSHLocalPortAccessible`, above `defer` changed to remove `pidFile`.
That’s why test fails with:
https://github.com/lima-vm/lima/actions/runs/17849303510/job/50754678943?pr=3979#step:9:165
> time="2025-09-19T05:30:22Z" level=fatal msg="expected status \"Running\", got \"Broken\" (maybe use `limactl stop -f`?)"

I think `filesToRemove` is not required.
I'll remove them.